src/ok.rs

```diff
@@ -815,7 +815,7 @@ fn snp_ioctl(test: SnpStatusTest) -> TestResult {
             }
         }
         SnpStatusTest::Rmp => {
-            if status.is_rmp_init == 1 {
+            if (status.is_rmp_init & PlatformInit::IS_RMP_INIT).bits() != 0 {
```

Comment:
Why are you checking both?

Reply:
I'm a bit unclear what you mean. This is a bitwise-AND, not a logical-AND?